templates/install-wp-tests.sh

```diff
@@ -11,7 +11,7 @@ DB_PASS=$3
 DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
 
-WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib}
+WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib/includes}
```

Comment:
I thought, `WP_UnitTestCase` should be placed in to `/tmp/wordpress-tests-lib/includes`.

https://github.com/miya0001/wp-cli/blob/master/templates/bootstrap.mustache#L15

And wp-tests-config.php should be placed parent directory of the `includes`.

```
$config_file_path = dirname( dirname( __FILE__ ) );
if ( ! file_exists( $config_file_path . '/wp-tests-config.php' ) ) { 
    // Support the config file from the root of the develop repository.
    if ( basename( $config_file_path ) === 'phpunit' && basename( dirname( $config_file_path ) ) === 'tests' )
        $config_file_path = dirname( dirname( $config_file_path ) );
}
$config_file_path .= '/wp-tests-config.php';
```
http://develop.svn.wordpress.org/trunk/tests/phpunit/includes/bootstrap.php.

Are there any problem?